README.md

```diff
@@ -138,9 +138,9 @@ Default:        false
 ```
 Type:           bool
 Valid Values:   true, false
-Default:        false
+Default:        true
 ```
-`allowNativePasswords=true` allows the usage of the mysql native password method.
+`allowNativePasswords=true` allows the usage of the mysql native password method, and to be `false` to disallow the usage of mysql native password method during auth switch process.
```

Comment:
Only the effect of `allowNativePasswords=false` has to be documented, as `true` is the default value and setting it manually wouldn't make sense as it doesn't change anything.